specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 23 Oct 2020 21:33:05 GMT",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-subscription-reads": "14909",
+        "x-ms-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b",
+        "x-ms-correlation-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.Synapse/workspaces/exampleWorkspaceName/libraryartifacts/exampleLibraryName1.jar",
+            "name": "exampleLibraryName1.jar",
+            "type": "Microsoft.Synapse/workspaces/libraryartifacts",
+            "properties": {
+              "name": "exampleLibraryName1.jar",
+              "path": "exampleWorkspace/libraries/exampleLibraryName1.jar",
+              "containerName": "exampleContainer",
+              "type": "jar",
+              "uploadedTimestamp": "2020-10-23T07:59:29.1573546+00:00",
+              "provisioningStatus": "Succeeded",
```

Comment:
>provisioningStatus [](start = 15, length = 18)

The standard name for this is `provisioningState`.

Reply:
This value is coming from partner service and the partner integrated with this API is using this with name only.